testproject/Assets/Tests/Manual/Scripts/ParentPlayerToInSceneNetworkObject.cs

```diff
@@ -0,0 +1,61 @@
+using UnityEngine.SceneManagement;
```

Comment:
Any chance there could be an automated test for this instead of only manual tests? Manual tests only catch things if someone remembers to run them.

Reply:
There are two tests in this PR:
Manual test: this script is used in that.
Integration Test: (see `ObjectParenting/ParentDynamicUnderInScenePlaced.cs` below)